R/semanticPage.R

```diff
@@ -58,47 +65,40 @@ get_range_component_dependencies <- function() { # nolint
 
 #' Get default semantic css
 #'
-#' @return path to default css semantic file
-get_default_semantic_theme <- function() {
-  if (getOption("shiny.minified", TRUE)) {
-    path <- file.path(get_cdn_path(), "semantic.min.css", fsep = "/")
-  } else {
-    path <- file.path(get_cdn_path(), "semantic.css", fsep = "/")
-  }
-  c(path)
-}
-
-#' Get default semantic js
+#' @param full_url define return output filename or full path. Default TRUE
 #'
-#' @return path to default js semantic file
-get_default_semantic_js <- function() {
-  if (getOption("shiny.minified", TRUE)) {
-    path <- file.path(get_cdn_path(), "semantic.min.js", fsep = "/")
-  } else {
-    path <- file.path(get_cdn_path(), "semantic.js", fsep = "/")
-  }
-  path
+#' @return path to default css semantic file or default filename
+get_default_semantic_theme <- function(full_url = TRUE) {
+  minfield <- ifelse(getOption("shiny.minified", TRUE), "min", "")
+  css_file <- paste("semantic", minfield, "css", sep = ".")
+  path <- file.path(get_cdn_path(), css_file, fsep = "/")
+  return(c(ifelse(full_url, path, css_file)))
 }
 
 #' Semantic theme path validator
 #'
 #' @param theme_css it can be either NULL, character with css path, or theme name
+#' @param full_url define return output filename or full path. Default TRUE
```

Comment:
defineS

Reply:
+ definition of `full_url` it's not super clear to me, can you rephrase? should be: `boolean flag that defines what is returned, either filename, or full path `?